spec.bs

```diff
@@ -1988,12 +2065,24 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
 <h4 id=navigation-changes>Actual navigation changes</h4>
 
 <div algorithm=source-snapshot-param-config>
-  Add a new [=struct/item=] to the [=source snapshot params=] [=struct=]:
+  Add 2 new [=struct/item=]s to the [=source snapshot params=] [=struct=]:
+
+  : <dfn for="source snapshot params">initiator fenced frame config instance</dfn>
+  :: a [=fenced frame config instance=] or null, initially null.
 
-  : <dfn for="source snapshot params">fenced frame config</dfn>
+  : <dfn for="source snapshot params">target fenced frame config</dfn>
```

Comment:
Now that we have two similar configs in this struct, can you add a more general note inside this `<div>`, not specific to either member, describing the difference between each of these and how they are used?